sample_factory/utils/huggingface_utils.py

```diff
@@ -0,0 +1,85 @@
+import os
+
+import cv2
+import numpy as np
+from huggingface_hub import HfApi, repocard, upload_file, upload_folder
+
+from sample_factory.utils.utils import experiment_dir
+
+
+def generate_replay_video(dir_path: str, frames: list, fps: int):
+    tmp_name = "/tmp/replay.mp4"
```

Comment:
Just in case, maybe use `project_tmp_dir` for this. I faced some issues with /tmp on clusters.
Just to be safe.